apps/examples/testcase/ta_tc/systemio/itc/itc_uart.c

```diff
@@ -259,6 +262,55 @@ void itc_systemio_iotbus_uart_set_flowcontrol_p(void)
 	TC_SUCCESS_RESULT();
 }
 
+/**
+* @testcase         itc_systemio_iotbus_uart_loopback_p
+* @brief            check the uart loopback mode support
+* @scenario         check the uart loopback mode support
+* @apicovered       iotbus_uart_write, iotbus_uart_read
+* @precondition     initializes uart_context
+* @postcondition    closes uart_context
+*/
+void itc_systemio_iotbus_uart_loopback_p(void)
+{
+	int ret = IOTBUS_ERROR_NONE;
+	char sz_input_text[BUF_LEN] = "UART LB TEST";
+	char sz_output_text[BUF_LEN];
+	iotbus_uart_context_h h_uart = iotbus_uart_init(DEVPATH);
+	TC_ASSERT_NEQ("iotbus_uart_init", h_uart, NULL);
+
+	// turn on loopback mode
+	ret = iotbus_uart_control_loopback(h_uart, LOOPBACK_ON);
+	if (ret < 0) {
+		iotbus_uart_control_loopback(h_uart, LOOPBACK_OFF);
+	}
+	TC_ASSERT_EQ_CLEANUP("iotbus_uart_control_loopback", ret, false, iotbus_uart_stop(h_uart));
```

Comment:
Here, the variable ret is of int type. So, you must not compare it to false. You must check if it is less than zero.